### PR TITLE
Adding Logo and Modify Readme.md

### PR DESCRIPTION
**Pull request by iqbalhood**
Hello Sir, I Have Added Logo on your project and modify the Readme.md file.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# jaguar_orm
+<img height="150" src="logo/Version1/Version1_hdpi.png">
```
Comment (jaumard) [inline README.md]: please add `alt="jaguar orm"` for SEO and accessibility purpose :)

**Comment by iqbalhood** [comment]
Hello Sir, I Have Finish Adding Alt for SEO and accessibility purpose

**Comment by jaumard** [comment]
Thanks @iqbalhood :) that was my only comment on my side ^^

**Comment by iqbalhood** [comment]
Okay Thank's. I hope the logo can be used soon :) 👍

**Comment by tejainece** [comment]
Looks fantastic mate! Great work!

**Pull request merged**